lib/ClangImporter/ImportDecl.cpp

```diff
@@ -6759,7 +6759,7 @@ SwiftDeclConverter::importSubscript(Decl *decl,
   // - There is no setter.
   bool associateWithSetter = !getterAndSetterInSameType && setter == decl;
   DeclContext *dc =
-      associateWithSetter ? setter->getDeclContext() : getter->getDeclContext();
+      associateWithSetter ? setter && setter->getDeclContext() : getter->getDeclContext();
```

Comment:
If `setter` was null then `associateWithSetter` would be `false`

Reply:
@theblixguy thanks for the review. Made some modifications.